recipes-extend/host-readme/files/Readme.md

```diff
@@ -0,0 +1,32 @@
+# RZ/G2L-SBC host folder
+
+## Description
+
+This directory contains files and tools used for managing and deploying images to the RZ/G2L-SBC platform. It includes build artifacts, manifests, test data, and documentation.
+
+## A top-level directory of host
+
+```
+host
+├── build
+│   ├── ${IMAGE_BASENAME}-rzg2l-sbc-20240717204209.rootfs.manifest                                       # Manifest file for the root filesystem
```

Comment:
As previous comment, IMAGE_BASENAME is too ambiguous, you can specify any target but need to put a note to explain.

Refer to: https://github.com/Renesas-SST/meta-renesas/blob/dunfell/rz-sbc-distro1-rc2-fix/meta-rzg2l/recipes-extend/sd-creator-linux/files/Readme.md?plain=1#L42